sql/glam-fenix-dev/glam_etl/firefox_desktop__view_clients_daily_scalar_aggregates_v1/view.sql

```diff
@@ -0,0 +1,10 @@
+-- view for firefox_desktop__view_clients_daily_scalar_aggregates_v1;
+-- View to union daily scalar aggregates with date partitioning
+CREATE OR REPLACE VIEW
+  `glam-fenix-dev.glam_etl.firefox_desktop__view_clients_daily_scalar_aggregates_v1`
```

Comment:
`glam-fenix-dev` doesn't apply here for desktop. I think the folder that the sql is generated into needs to be changed as well.

Reply:
good point. I will work on this